src/views/confirm.njk

```diff
@@ -0,0 +1,576 @@
+  Other / Equivalent Deer Stalking Certificate
+  {% endif %}
+{% endset -%}
+
+{% set qualificationPaths %}
+  {% if model.qualificationHeld === 'dsc1' %}
+  /dsc1-details
+  {% elif model.qualificationHeld === 'dsc2' %}
+  /dsc2-details
+  {% else %}
+  /other-qualification-details
+  {% endif %}
+{% endset -%}
+
+{% set qualificationNumberIds %}
+  {% if model.qualificationHeld === 'dsc1' %}
+  #dsc1-number
+  {% elif model.qualificationHeld === 'dsc2' %}
+  #dsc2-number
+  {% else %}
+  #other-name
+  {% endif %}
+{% endset -%}
+
+{% set qualificationDateIds %}
+  {% if model.qualificationHeld === 'dsc1' %}
+  #dsc1-date
+  {% elif model.qualificationHeld === 'dsc2' %}
+  #dsc2-date
+  {% else %}
+  #other-date
+  {% endif %}
+{% endset -%}
+
+  <form method="post">
+    <h1 class="govuk-heading-xl">
+      Check your answers before sending your application
+    </h1>
+
+    <h2 class="govuk-heading-m">
+      Application details
+    </h2>
+
+    {{ govukSummaryList({
+      rows: [
+        {
+          key: {
+            text: "Criminal Convictions"
+          },
+          value: {
+            text: "I have not been convicted of a wildlife crime." if model.conviction === false
+          },
+          actions: {
+            items: [
+              {
+                href: pathPrefix + "/conviction#conviction",
+                text: "Change",
+                visuallyHiddenText: "convictions"
+              }
+            ]
+          }
+        },
+        {
+          key: {
+            text: "Firearms Certificate Number"
+          },
+          value: {
+            text: model.certificateNumber
+          },
+          actions: {
+            items: [
+              {
+                href: pathPrefix + "/firearm#certificate-number",
+                text: "Change",
+                visuallyHiddenText: "firearm certificate number"
+              }
+            ]
+          }
+        },
+        {
+          key: {
+            text: "Firearms Certificate Date Obtained"
+          },
+          value: {
+            text:  model.certificateIssuedDay + '/' + model.certificateIssuedMonth + '/' + model.certificateIssuedYear
+          },
+          actions: {
+            items: [
+              {
+                href: pathPrefix + "/firearm#certificate-issued",
+                text: "Change",
+                visuallyHiddenText: "firearm certificate issued date"
+              }
+            ]
+          }
+        },
+        {
+          key: {
+            text: "Qualification held"
+          },
+          value: {
+            text: qualificationStrings
+          },
+          actions: {
+            items: [
+              {
+                href: pathPrefix + "/qualification#qualificationHeld",
+                text: "Change",
+                visuallyHiddenText: "qualification held"
+              }
+            ]
+          }
+        },
+        {
+          key: {
+            text: "Qualification Number / Name"
```

Comment:
This could be 'templated' up the top like the other ones you're doing, so it's either 'Qualification Number' or 'Qualification Name'.

Reply:
Resolved in 61e0396aa1bd6ec9f0cb3b1696f58572f10396db.